src/App.css

```diff
@@ -12,7 +12,22 @@
     animation: App-logo-spin infinite 20s linear;
   }
 }
+.main-area{
```

Comment:
The app utilizes Tailwind instead of vanilla CSS, to ensure the app is optimized and in line with the rest of the code, please rewrite using Tailwind.